src/lib/Routes.ts

```diff
@@ -0,0 +1,66 @@
+export enum Routes {
+    Dashboard = "/dashboard",
+    Notification = "/notification",
+    Attendance = "/attendance",
+    MarkAttendance = "/mark-attendance",
+    Fund = "/fund",
+    Member = "/member",
+    AddMember = "/add-member",
+    ShiftIncharge = "/shift-incharge",
+    AddShiftIncharge = "/add-shift-incharge",
```

Comment:
You cannot add a shift-incharge. @ghulamhafeez assist here. This can only be updated from update user screen.

Reply:
Fixed